src/main/java/org/alliancegenome/curation_api/model/ingest/dto/fms/WhereExpressedDTO.java

```diff
@@ -0,0 +1,17 @@
+package org.alliancegenome.curation_api.model.ingest.dto.fms;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import org.alliancegenome.curation_api.model.ingest.dto.base.BaseDTO;
+
+import java.util.List;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class WhereExpressedDTO extends BaseDTO {
+	private String whereExpressedStatement;
+	private String anatomicalStructureTermId;
+	private String cellularComponentTermId;
+	private List<UberonSlimTermDTO> anatomicalStructureUberonSlimTermIds;
```

Comment:
I am seeing a bunch of extra fields in the json file. Are these going to be ignored?
`cellularComponentQualifierTermId`
`anatomicalStructureTermId`
`anatomicalStructureQualifierTermId`
`anatomicalSubStructureTermId`
`anatomicalSubStructureUberonSlimTermIds`
`anatomicalSubStructureQualifierTermId`

Reply:
In this first pass, they are ignored

Reply:
but its easy enough to add them and ignore them later

Reply:
Added